tests/vm.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { defineComponent, ref } from 'vue'
+
+import { mount } from '../src'
+
+describe('vm', () => {
+  it('returns the component vm', () => {
+    const Component = defineComponent({
+      template: '<div>{{ msg }}</div>',
+      setup() {
+        const msg = 'hello'
+        const isEnabled = ref(true)
+        const toggle = () => (isEnabled.value = !isEnabled.value)
```

Comment:
Toggle is not necessary here.

Reply:
👍 Removed (it was a leftover of a more in-depth test for #64 )